plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -80,6 +107,25 @@ func defaultPredicates() sets.String {
 		),
 		// Fit is determined by the presence of the Host parameter and a string match
 		factory.RegisterFitPredicate("HostName", predicates.PodFitsHost),
+
+		// Fit is determined by whether or not there would be too many AWS EBS volumes attached to the node
+		factory.RegisterFitPredicateFactory(
+			"MaxEBSVolumeCount",
+			func(args factory.PluginFactoryArgs) algorithm.FitPredicate {
+				// TODO: allow for generically parameterized scheduler predicates, because this is a bit ugly
+				maxVols := getMaxVols(DefaultMaxEBSVolumes)
+				return predicates.NewMaxPDVolumeCountPredicate(predicates.EBSVolumeFilter, maxVols, args.PVInfo, args.PVCInfo)
+			},
+		),
+
+		factory.RegisterFitPredicateFactory(
```

Comment:
Add comment like you did for the other one: "// Fit is determined by whether or not there would be too many GCE PD volumes attached to the node"